rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -0,0 +1,247 @@
+#
+# Copyright (c) 2019 Connexta, LLC
+#
+# Released under the GNU Lesser General Public License version 3; see
+# https://www.gnu.org/licenses/lgpl-3.0.html
+#
+
+openapi: 3.0.2
+
+info:
+  version: ${project.version}
+  title: Product API
+  description: >
+    The Product API allows clients to create and retrieve Product resources
+  contact:
+    name: Connexta
+  license:
+    name: GNU Lesser General Public License
+    url: 'http://www.gnu.org/licenses/lgpl.html'
+
+servers:
+  - url: ${rest.server.url}
+
+paths:
+  /mis/product:
+    post:
+      summary: Create a Product
+      description: >
+        Clients send a file attachment to create a Prodcut resource.
+      operationId: productPOST
```

Comment:
✏️ this translates to the API method name, so maybe `createProduct`?

Reply:
I changed this to match the name of the method in the current StoreApi class of the ion-store repo.